src/main/java/org/vafer/jdeb/changes/TextfileChangesProvider.java

```diff
@@ -46,15 +47,24 @@ public final class TextfileChangesProvider implements ChangesProvider {
 
     private final ChangeSet[] changeSets;
 
-    private DateFormat fmt = new SimpleDateFormat("HH:mm dd.MM.yyyy");
+    private final DateFormat fmt;
```

Comment:
Why did you move this into the constructor? Just curious.

Reply:
Because we conditionally call `fmt.setTimeZone(TimeZone.getTimeZone("UTC"))` depending upon whether or not the `outputTimestampMs` argument is null,  it seemed more logical to keep the initialization of `fmt` near to this call.

Additionally, since `fmt` is not mutated outside of the constructor, it seemed prudent to declare it `final` as well.